LLM/language_model.py

```diff
@@ -18,6 +18,15 @@
 console = Console()
 
 
+WHISPER_LANGUAGE_TO_LLM_LANGUAGE = {
+    "<|en|>": "english",
+    "<|fr|>": "french",
+    "<|es|>": "spanish",
+    "<|zh|>": "chinese",
+    "<|ja|>": "japanese",
+    "<|ko|>": "korean",
+}
+
```

Comment:
This would break the logic. The reason for having a smaller dictionary is to force whispers to re-infer if it thinks that the spoken language is one we don't support yet. This considerably improves the user experience, because it doesn't happen that whisper thinks you are speaking Italian and then the LLM answers Italian, and then the voice speaks the Italian text with an English voice.